### PR TITLE
Add wrapper class for overdue appointment sections

### PR DESCRIPTION
**Pull request by msasikanth**
https://app.shortcut.com/simpledotorg/story/8436/add-wrapper-class-for-overdue-appointment-sections

### CODE REVIEW

**Comment by msasikanth** [comment]
Current dependencies on/for this PR:
* master
  * **PR #3906** <a href="https://app.graphite.dev/github/pr/simpledotorg/simple-android/3906" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/simpledotorg/simple-android/3906?utm_source=stack-comment).

**Review by Honey14 — APPROVED** [review]
LGTM

**Comment by msasikanth** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/simpledotorg/simple-android/3906?utm_source=stack-comment).


Job Reference: ZDCa9ltwbvB0lfaAvhSM

**Pull request merged**
